MusicNotationKit/MusicNotationKit/Staff.swift

```diff
@@ -42,19 +42,35 @@ public struct Staff {
      
```

Comment:
If there is a repeat, the insert index can still fall at the start of a repeated measure (equivalent to prepend before repeated measure), or at the end of the repeated measure (append after repeated measure).  Any inserts within the repeated measure require inRepeat set to true, which allows the repeated measure to include the new measure.  In this case the number of measures will increase by repeatCount of the MeasureRepeat instance .